products/memgraph/upcoming/tutorials/understanding-music-with-modules.md

```diff
@@ -0,0 +1,477 @@
+we'll see how easy it is to extend the capabilities of Memgraph using Python.
+It will also show you how to use one of the most popular Python libraries for graphs,
+called [NetworkX](https://networkx.github.io/), which contains an insane amount
+of functions and algorithms for the graphs.
+
+### Data model
+
+Social graphs is a relatively recent term. Simply said, it's a representation of
+a social network. Social networks appear in various sites, e.g., Youtube,
+which is primarily a site for watching videos, can be classified as a social
+network. For this tutorial, we'll use data consisting of users of the music
+streaming platform called [Deezer](https://www.deezer.com/).
+
+The data consists of around 50k Deezer users from Croatia, but we prepared a
+subset of the dataset only composed of 2k users. Each user is defined by id
+and a list of genres he loved. The edges represent the mutual friendship between
+the users. You can find a more detailed explanation of the dataset on the
+[github](https://github.com/benedekrozemberczki/datasets#deezer-social-networks)
+alongside many more similar datasets kindly provided by the same authors.
+
+### Importing the Snapshot and defining a directory with user modules
+
+We have prepared a database snapshot for this example, so the user can easily
+import it when starting Memgraph using the `--data-directory` option.
+
+Also, we need to set up the directory from which the Memgraph will search for
+the query modules by changing the `--query-modules-directory` flag in the main
+configuration file(`/etc/memgraph/memgraph.conf`) or by supplying it as a
+command-line parameter using the directory containing our modules as the value.
+For more detailed explanation take a look at
+[How to Use and Implement Query Modules](../how_to_guides/use-and-implement-query-modules.md).
+
+```plaintext
+/usr/lib/memgraph/memgraph --data-directory /usr/share/memgraph/examples/Music \
+ --storage-snapshot-interval-sec=0 --storage-wal-enabled=false \
+ --storage-snapshot-on-exit=false --storage-properties-on-edges=true
+ --query-modules-directory=/modules
+```
+
+When using Memgraph installed from DEB or RPM package, the currently running
+Memgraph server may need to be stopped before importing the example. The user
+can do so using the following command:
+
+```plaintext
+systemctl stop memgraph
+```
+
+When using Docker, the example can be imported with the following command
+(notice how we need to mount the query module directory first):
+
+```plaintext
+docker run -p 7687:7687 \
+  -v mg_lib:/var/lib/memgraph -v mg_log:/var/log/memgraph -v mg_etc:/etc/memgraph \
+  -v $(pwd)/modules:/modules \
+  memgraph --data-directory /usr/share/memgraph/examples/Music \
+  --storage-snapshot-interval-sec=0 --storage-wal-enabled=false \
+  --storage-snapshot-on-exit=false --storage-properties-on-edges=true
+  --query-modules-directory=/path/to/mount/of/modules/directory
```

Comment:
Isn't the correct path `--query-modules-directory=/modules`?

Reply:
Yes it is